PocketKit/Sources/PocketKit/SceneTracker.swift

```diff
@@ -7,7 +7,8 @@ import Analytics
 import SwiftUI
 import Combine
 
-class SceneTracker {
+@MainActor
```

Comment:
Does this need MainActor, I don't see anything that needs the main thread?

Reply:
in that class, we listen for two notifications, that generate the following warnings:
`Main actor-isolated class property 'didActivateNotification' can not be referenced from a non-isolated context; this is an error in Swift 6`

`Main actor-isolated class property 'didEnterBackgroundNotification' can not be referenced from a non-isolated context; this is an error in Swift 6`

if we add `@MainActor` to the specific methods, we then have to also add it to `init`, which basically means adding the type itself